scripts/generateAssetData/utils/portals.ts

```diff
@@ -78,6 +114,7 @@ export const fetchPortalsTokens = async (
 }
 
 export const getPortalTokens = async (nativeAsset: Asset): Promise<Asset[]> => {
+  const portalsPlatforms = await fetchPortalsPlatforms()
```

Comment:
Technically one call per EVM chain when we could do with only a single one for alls but meh - could potentially pass this around but was quite messy to do so all the way down from `generateAssetData`.
Not a big deal IMO to have 7 additional calls, we don't really run this often.